IyiKuyoro/TalesByYomi
# #161782642 setup eslint

### What does this PR do
This PR setsup eslint as the default linting tool

### Tasks completed
1. Installed eslint as a dev dependency
2. Add the eslint config file
3. Setup eslint to use the airbnb style guide

### How can this be manually tested
1. Clone this repo
2. Checkout the `ch-setup-eslint-#161782642` branch
3. Ensure you have the eslint plugin installed on your browser
4. Type in any remove one of the trailing commas in an object and watch eslint flag that as an error

### Relevant PT stories
#161782642
